modules/vaos/spec/request/appointment_requests_request_spec.rb

```diff
@@ -84,7 +84,7 @@
 
           expect(response).to have_http_status(:success)
           expect(response.body).to be_a(String)
-          expect(response).to match_response_schema('vaos/appointment_requests')
+          expect(response).to match_response_schema('vaos/appointment_requests', strict: false)
```

Comment:
hmm, is it necessary because the object is sometimes nil and sometimes a hash? I wonder if this will evaluate to with strict: false if you change nil valued ones to an empty hash maybe?